roles/api-environment/templates/wait-for-container.sh.j2

```diff
@@ -0,0 +1,14 @@
+docker exec -i $container_name /bin/bash <<EOF
+timer=0
+timeout={{ container_timeout_seconds }}
+echo "waiting for container to be healthy..."
+until $health_command > /dev/null 2>&1; do
+    if [ \$timer -ge \$timeout ]; then
+        echo "container not healthy after \$timeout seconds"
+        exit 1
+    fi
+    sleep 1
+    timer=\$((\$timer + 1))
+done
+echo "container healthy after \$timer seconds"
+EOF
```

Comment:
Not a big deal, but in the future, we should probably use `git mv` to rename the file so that we can still keep this file changed history and track the issue easily.

Reply:
I did, but after I made some changes to it and added those changes, it changed the status to the old file being deleted and the new one being added. This has happened before too and I'm not sure what to do. Have you ran into this before?

Reply:
I tried rebasing and inserting a commit that renamed the file. From my local history, the changes seemed to look correct (`gdst` is just `git --no-pager diff --stat`):

```shell
$ gdst head~2..head^
 roles/api-environment/templates/{timeout-delay.sh.j2 => wait-for-container.sh.j2} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)

$ gdst head^
 roles/api-environment/defaults/main.yml                                   |  2 +-
 roles/api-environment/templates/dropwizard-api/start-container.sh.j2      |  5 +++--
 roles/api-environment/templates/express-api/start-express-container.sh.j2 |  5 +++--
 roles/api-environment/templates/wait-for-container.sh.j2                  | 24 ++++++++++++++----------
 4 files changed, 21 insertions(+), 15 deletions(-)
```

but the diff on github still says it's deleted and a new file was added. Not sure what else to do, haha. Any ideas?

Reply:
Changing the base to another branch and then back again didn't work either.